plugins/platform-ui/src/routes.ts

```diff
@@ -112,22 +112,22 @@ export class Router<T> implements ApplicationRouter<T> {
   ) {
     this.pattern = pattern
     this.parentRouter = parent
-    if (defaults) {
+    if (defaults != null) {
```

Comment:
I suppose it should be `!==` and `undefined`. There are a lot of same issues within file.

Reply:
Thanks. fixed.